backend/blueprint/api/models/file.py

```diff
@@ -4,8 +4,8 @@
 class File(db.Model):
     __tablename__ = 'File'
 
-    filename = db.Column(db.String(100), NULLABLE=False)
-    filetype = db.Coloumn(db.string(100), NULLABLE=False)
+    filename = db.Column(db.String(100), nullable=False, primary_key=True)
```

Comment:
How can you set filename as primary key? How are you making sure that it will be unique?

Reply:
uuid.uuid4 and uuid.uuid1 provide unique names always.
I already had it discussed with @yashLadha and @ParthS007 since they raised the same doubt

Reply:
okay

Reply:
and we are using `uuid.uuid4` for generating the names. therefore it will always be unique.